internal/component/otelcol/processor/k8sattributes/types.go

```diff
@@ -157,6 +157,20 @@ func (args ExcludeConfig) convert() map[string]interface{} {
 	for _, pod := range args.Pods {
 		pods = append(pods, pod.convert())
 	}
+
+	// These are default excludes from upstream opentelemetry-collector-contrib
+	// Source: https://github.com/open-telemetry/opentelemetry-collector-contrib/blame/main/processor/k8sattributesprocessor/factory.go#L21
+	if len(pods) == 0 {
+		pods = append(pods,
+			map[string]interface{}{
+				"name": "jaeger-agent",
+			},
+			map[string]interface{}{
+				"name": "jaeger-collector",
+			},
+		)
+	}
```

Comment:
I think this works, but it doesn't really align with how we handle defaulting in other components. You should be able to implement the river.Defaulter interface on the [Arguments type](https://github.com/grafana/agent/blob/4cd7f745d9411130abc783ec3c82674eba9cb6e6/internal/component/otelcol/processor/k8sattributes/k8sattributes.go#L34) to set defaults like so:

```go 
func (args *Arguments) SetToDefault() {
  args.Exclude = ExcludeConfig{
    Pods: []ExcludePodConfig{
      {Name: "jaeger-agent"},
      {Name: "jaeger-collector"}, 
    },
  }
}
```

Let me know what you think :)

Reply:
I implemented it in previous manner as I think `SetToDefault()` would only be tested via E2E/Integration. Boy that was horribly wrong :D.

Thank you for informing me of this pattern. I'll remember well next time for existing/new components!

Btw, I've made the proposed change in the HEAD :D